operator/runner.go

```diff
@@ -214,17 +238,27 @@ func (s *Runner) Run(ctx context.Context, provider app.Provider) error {
 	}
 
 	// Metrics
-	if s.metricsServer != nil {
-		err = s.metricsServer.RegisterCollectors(runner.PrometheusCollectors()...)
+	if s.metricsExporter != nil {
+		err = s.metricsExporter.RegisterCollectors(runner.PrometheusCollectors()...)
 		if err != nil {
 			return err
 		}
-		runner.AddRunnable(s.metricsServer)
+
+		s.server.RegisterMetricsHandler(s.metricsExporter.HTTPHandler())
 	}
 
+	// Add Metrics+Health cumulative server runnable, healthcheck based on "running" should
+	// be passed down to the serverRunner
+	runner.AddRunnable(s.serverRunner)
```

Comment:
I think here we also want to run through `runner.HealthChecks()` and register all of them with `s.serverRunner`.